yale_medical_config.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require 'traject_plus'
+require 'dlme_json_resource_writer'
+require 'macros/dlme'
+require 'macros/csv'
+extend Macros::DLME
+extend Macros::Csv
+extend TrajectPlus::Macros
+extend TrajectPlus::Macros::Csv
+
+settings do
+  provide 'writer_class_name', 'DlmeJsonResourceWriter'
+  provide 'reader_class_name', 'TrajectPlus::CsvReader'
+end
+
+# CHO required
+to_field 'id', normalize_prefixed_id('OrbisBibID'), strip
+to_field 'cho_title', column('Title'), strip
+
+# CHO other
+to_field 'cho_alternative', column('Variant Titles', split: '|'), strip
+to_field 'cho_coverage', column('Subject, geographic code'), strip
```

Comment:
I'd take this out - in records where this is present, it's just indicating that the volume is physically held in New Haven, Connecticut.